File: Phase6/backend/src/Users/fullstandard.py
from .user import User

class FullStandard(User):
    def __init__(self, username, password, credit):
        super().__init__(username, "FS", password, credit)
        
    def __str__(self):
        return f"FullStandard({self.username},{self.privilege},{self.password},{self.credit})"

File: Phase4/frontend/src/Users/fullstandard.cpp
#include "../include/Users/fullstandard.h"

// Constructs FullStandard User with "FS" privileges
FullStandard::FullStandard(const std::string& username, const std::string& password, const float& credit) 
    : User(username, "FS", password, credit) {}